src/lib/bootloader/bootloader_factory.rb

```diff
@@ -59,8 +59,9 @@ def supported_names
           system_bl = nil
         end
         ret = system_bl ? [system.name] : [] # use current as first
-        ret << "grub2" unless Yast::Arch.aarch64 # grub2 everywhere except aarch64
-        ret << "grub2-efi" if Yast::Arch.x86_64 || Yast::Arch.aarch64
+        # grub2 everywhere except aarch64 or riscv64
+        ret << "grub2" unless Yast::Arch.aarch64 || Yast::Arch.riscv64
```

Comment:
I think here it also makes sense to use `Systeminfo.efi_mandatory?` I think missing Arch.arm is just bug

Reply:
done